test/suite/test_schema08.py

```diff
@@ -158,8 +154,8 @@ def run_recovery(self, uri, suburi):
     # Test that creating and dropping tables does not write individual
     # log records.
     def test_schema08_create(self):
-        if self.is_tiered_scenario() and (self.uri == 'lsm:' or self.uri == 'file:' or self.schema_ops == 'rename'):
-            self.skipTest('Tiered storage does not support LSM or file URIs, and also does not support the rename operation.')
+        if self.is_tiered_scenario() and (self.uri == 'lsm:' or self.uri == 'file:'):
+            self.skipTest('Tiered storage does not support LSM and file URIs.')
```

Comment:
I think the original `or` was correct. But you and I have some shared history with grammar arguments...

Reply:
Haha I am not fussed about this, just thought it read better - it means the same thing. Updated here [4651212](https://github.com/wiredtiger/wiredtiger/pull/10282/commits/4651212791deac3577df5658fab5aa3c0f6f45c4)